core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -32,11 +32,26 @@ oppia.factory('UrlInterpolationService', [
       }
     };
 
-    var getCachePrefixedUrl = function(resourcePath) {
+    var getPrefixedUrl = function(resourcePath) {
       validateResourcePath(resourcePath);
       return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
     };
 
+    var getUrlWithSlug = function(resourcePath) {
+      validateResourcePath(resourcePath);
+      stripedPath = resourcePath.substring(1);
+      if (GLOBALS.MINIFICATION || !GLOBALS.DEV_MODE) {
+        if (!angular.isUndefined(hashes[stripedPath])) {
+          hash = hashes[stripedPath];
+          splitedPath = resourcePath.split('.');
```

Comment:
I know, but this isn't actually obvious -- the reader has to think a bit about what's going on.

Hence the suggestion to do the find-from-right a little more explicitly using e.g. lastIndexOf().